mx.jruby/mx_jruby.py

```diff
@@ -63,41 +63,36 @@ def getResults(self):
 
 # Commands
 
-def extractArguments(cli_args):
+def extractArguments(args):
     vmArgs = []
     rubyArgs = []
     classpath = []
     print_command = False
 
-    jruby_opts = os.environ.get('JRUBY_OPTS')
-    if jruby_opts:
-        jruby_opts = jruby_opts.split(' ')
```

Comment:
`RUBYOPT` and `TRUFFLEOPT` are both handled entirely in Java code, which is a very good thing.

We should consider `JAVA_OPT` here perhaps, though.